### PR TITLE
[Docs] Cleanup removed methods and style fixes

### PR DESCRIPTION
**Pull request by enen92**
## Description
Remove python documentation for functions that were completely removed in v17 and v12. If this is not done they endup annotating wrong methods. Also fixes some display issues (at least two "spaces" are required before a line break). Also hides a method that was incorrectly being showed as part of the api (Listitem.fromString).

## Motivation and Context
Complete docs

## How Has This Been Tested?
compiled docs and tested

## Types of change

- [x] **Bug fix** (non-breaking change which fixes an issue)
- [x] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [x] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [x] My change requires a change to the documentation, either Doxygen or wiki
- [x] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [x] All new and existing tests passed

### CODE REVIEW

File: xbmc/addons/kodi-addon-dev-kit/doxygen/Modules/modules_python.dox [hunk 1]
```diff
@@ -190,4 +200,4 @@ web applications or frameworks for the Python programming language.
 - @subpage python_v13
 - @subpage python_v12
 
-+*/
++*/
```
Comment (garbear) [inline xbmc/addons/kodi-addon-dev-kit/doxygen/Modules/modules_python.dox]: Missing newline at end of file

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by enen92** [comment]
@MartijnKaijser please hold on merging this one, maybe 
https://github.com/xbmc/xbmc/pull/14724/files#diff-8fbc70938c5182e2dff98103a539108bL1163
is wrong. I need to read a bit to the history to see what that was.

**Comment by MartijnKaijser** [comment]
@enen92 http://mirrors.kodi.tv/docs/python-docs/13.0-gotham/xbmcgui.html#ListItem-addContextMenuItems

**Comment by enen92** [comment]
Just for reference @DaVukovic : https://github.com/xbmc/xbmc/commit/8e8ab97f9e07d88bc8ad0aa20e01b075d0eb21bf#diff-8fbc70938c5182e2dff98103a539108b

This was "wrongly" introduced when we first had changelogs available for the python documentation

**Pull request merged**